### DISCUSSION TITLE
Client can not receive data from Server

### DISCUSSION THREAD

**Discussion by HieuJoe** [jchristn/SuperSimpleTcp#149] [Q&A]
Hello everyone.
I am a beginner, I am using SuperSimpleTcp following the sample. But the Client can not receive data from Server. 
Please help me. Thanks everyone

![image](https://user-images.githubusercontent.com/112363990/187115148-c28555e9-2377-4951-966c-fe53d57f49ce.png)

![image](https://user-images.githubusercontent.com/112363990/187114905-c93db008-654f-4adf-b96d-4528069270c9.png)

**Comment by jchristn** [comment]
Hi, on your server Send call, the first parameter should be the IP:port string you receive in ConnectionEventArgs when the client connects.

**Reply by jchristn** [reply]
i.e. you would call ```Send("127.0.0.1:58799", ...``` based on your screenshots
